client/src/pages/admin/courses.tsx

```diff
@@ -239,13 +239,37 @@ function Page() {
           </Select>
         </Form.Item>
 
-        <Form.Item
-          name="minStudentsPerMentor"
-          label="Minimum Students per Mentor"
-          rules={[{ min: 1, type: 'integer', message: 'Ensure that the input, if provided, is a positive integer.' }]}
-        >
-          <InputNumber step={1} />
-        </Form.Item>
+        <Row>
+          <Col span={12}>
+            <Form.Item
+              name="minStudentsPerMentor"
+              label="Minimum Students per Mentor"
+              rules={[
+                { min: 1, type: 'integer', message: 'Ensure that the input, if provided, is a positive integer.' },
+              ]}
+            >
+              <InputNumber step={1} />
+            </Form.Item>
+          </Col>
+          <Col span={12}>
+            <Form.Item
+              name="certificateThreshold"
+              label="Certificate Threshold"
+              tooltip="Minimum score percentage required for students to qualify for a certificate."
+              rules={[
+                {
+                  required: true,
```

Comment:
We discussed with @dzmitry-varabei that the default value is 70%, and it is also stated as such in our documentation. I'm not sure we need a 0 here because we don't have a course where we issue certificates to all who register.